workspaces/web/src/routes/development.svelte

```diff
@@ -3,6 +3,8 @@
   import Mascot from "../components/Mascot"
   import TwitterButton from "../components/TwitterButton"
   import GitHubButton from "../components/GitHubButton"
+  import Tag from "lluis/Tag"
+  import Tag from "lluis/Tag"
```

Comment:
This is a duplicate declaration, which is also making the CI fail:

https://kantord.semaphoreci.com/jobs/8d0c9eb7-efd9-437c-9b47-8286f390701e